### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by tommi-t-nguyen**
### What (if any) features are you implementing?
-testing for landing features and mood page
### What (if anything) did you refactor?
-n/a
### Were there any issues that arose?
-n/a

### Refactor checklist
- [ ] Check variable and parameter names: are they descriptive and unique?
- [ ] Is the code DRY? SRP?
- [ ] Is it easy for others to read your code? (explanatory comments needed?)
- [ ] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ ] Are commit messages formatted correctly?

### CODE REVIEW

File: cypress/fixtures/kanyeQuote.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (anewb87) [inline cypress/fixtures/kanyeQuote.json:1]: Stub! That! Data!

File: cypress/integration/features-page-test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+describe('Features page', () => {
```
Comment (anewb87) [inline cypress/integration/features-page-test.js:1]: Passing all the tests. Woot. Woot.

File: src/Pages/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ const Dashboard = () => {
         <article>Stat 2</article>
         <article>Stat 3</article>
       </section>
-      <section className='mood-section'>
+      <section className='mood-section' data-testid='mood-section'>
```
Comment (anewb87) [inline src/Pages/Dashboard/Dashboard.js:25]: Was told today in our job shadow that this is best practice.

**Review by anewb87 — APPROVED** [review]
3 pages down, 2 to go

File: cypress/integration/landing-page-test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  it('Should show us a kanye quote', () => {
+    cy.get('h1')
+      .should('have.text', 'You may be talented, but you’re not Kanye West.')
+  });
+  it('Should have a button that appears in 3 seconds that, upon click, takes you to the features page', () => {
+    cy.wait(3500)
+    cy.get('[data-testid=landing-button]')
+      .should('be.visible')
+      .click()
+    cy.server()
```
Comment (kavakai) [inline cypress/integration/landing-page-test.js:20]: I like these server > route cypress keywords

File: cypress/integration/mood-page-test.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  beforeEach(() =>{
+    cy.visit('http://localhost:3000/mood')
+  });
+  it('Should have a kanye face', () => {
+    cy.get('img')
+      .should('have.attr', 'src', '/static/media/kanye3.42c9905149cdaf49e335.png')
+  });
+  it("Should have a slider that changes the kanye value and face", () => {
+
+    const currentValue  = 3;
```
Comment (kavakai) [inline cypress/integration/mood-page-test.js:11]: 🤩

**Review by kavakai — APPROVED** [review]

**Review by gjtrecny — APPROVED** [review]

**Pull request merged**